packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -93,6 +103,19 @@ export default util.createRule<Options, MessageId>({
         'An explicit null check is required.',
       noStrictNullCheck:
         'This rule requires the `strictNullChecks` compiler option to be turned on to function correctly.',
+
+      conditionFixDefaultEmptyString:
+        'Explicitly treat nullish value the same as an empty string',
+      conditionFixDefaultZero: 'Explicitly treat nullish value the same as 0',
+      conditionFixCompareNullish:
+        'Change condition to check for null/undefined',
+      conditionFixCastBoolean: 'Explicitly cast value to a boolean',
+      conditionFixCompareStringLength:
+        "Change condition to check string's length",
+      conditionFixCompareEmptyString:
+        'Change condition to check for empty string',
+      conditionFixCompareZero: 'Change condition to check for 0',
+      conditionFixCompareNaN: 'Change condition to check for NaN',
```

Comment:
suggestion for the suggestion text - show what the fixed code would look like to make it clearer.

EG:
```suggestion
      conditionFixDefaultEmptyString:
        'Explicitly treat nullish value the same as an empty string (`value ?? ''`)',
      conditionFixDefaultZero:
        'Explicitly treat nullish value the same as 0 (`value ?? 0`)',
      conditionFixCompareNullish:
        'Change condition to check for null/undefined (`value != null`)',
      conditionFixCastBoolean:
        'Explicitly cast value to a boolean (`Boolean(value)`)',
      conditionFixCompareStringLength:
        "Change condition to check string's length (`value.length !== 0`)",
      conditionFixCompareEmptyString:
        'Change condition to check for empty string (`value !== ""`)',
      conditionFixCompareZero:
        'Change condition to check for 0 (`value !== 0`)',
      conditionFixCompareNaN: 'Change condition to check for NaN (`!Number.isNaN(value)`)',
```